src/commands/remove.js

```diff
@@ -7,20 +7,34 @@ const {userConfig, isLoggedIn, logout} = require('../config');
 
 exports.command = ['remove <id>', 'rm <id>'];
 exports.describe = 'remove active deployment';
-exports.builder = {};
-exports.handler = async ({id}) => {
-  if (!isLoggedIn()) {
+exports.builder = {
+  token: {
+    alias: 't',
+    description: 'Deployment token to be used for authentication',
+  },
+};
+exports.handler = async (args = {}) => {
+  const deployToken = args.token;
+  const id = args.id;
+
+  if (!deployToken && !isLoggedIn()) {
     return;
   }
 
   console.log(chalk.bold('Removing deployment:'), id);
 
   // services request url
   const remoteUrl = `${userConfig.endpoint}/remove/${encodeURIComponent(id)}`;
+  let authToken = userConfig.token;
+
+  if (deployToken) {
+    authToken = deployToken;
+    console.log('\nDeploying using given token..');
```

Comment:
Please fix the logging message here (we're not deploying, are we?)

Reply:
Is using "Undeploying ..." ok ?

Reply:
`Removing` is the terminology that was used so far, I'd prefer to be consistent